### ISSUE TITLE
[FEATURE] Topology hiding module also hide domains from and to headers

### ISSUE THREAD

**Issue by Otoru** [OpenSIPS/opensips#2430]
**Implementation**
  - Component: `topology hiding` module

It would be nice if the topology hiding module also hid domains from and to headers.

**Describe alternatives you've considered**

I'm hiding these fields manually

**Comment by bogdan-iancu** [comment]
Why not using the [uac_replace_from/to()](https://opensips.org/html/docs/modules/3.1.x/uac.html#func_uac_replace_from)   functions in conjunction with the topp hiding module?

**Comment by Otoru** [comment]
Yes, I am doing this manually at the moment, but it would be nice to not have to worry about it when using the topology hiding module.

**Comment by bogdan-iancu** [comment]
doing the `uac_replace_from()` only once (at initial INVITE) is sufficient - the same as doing TH. I do not see what extra manual ops you need to do.

**Comment by Otoru** [comment]
I understand that it is only necessary to deal with this in the initial INVITE and that is what I do today.

Giving a context, in one of my environments I have a proxy with multiple interfaces (socket's) and the drouting selects the source socket and with the TH the SIP message is hidding, except for FROM/TO header that I need to write manually.

It would be nice not to have to worry about that.

**Comment by Otoru** [comment]
I must be consulting the wrong sources. I will close the issue because you have already given me a good direction. Thank you!

**Comment by farmorg** [comment]
I would really like to see this too. I have some reasonably complex routing going on in a very similar fashion to @Otoru using TH and I also have to use uac_replace_from()/to().

This would make for much a cleaner script :)
